server/server/search/view.py

```diff
@@ -77,7 +78,8 @@ def get(self):
 
         if query is None:
             return json.dumps({'error': '\'query\' param is required'}), 422
-
+            
+        results = {'total': 0, 'hits': []}
```

Comment:
Variable not used

Reply:
Variable used now. Elasticsearch is a good deal more likely to fail to return results than ArangoDB so I'm writing the code with the possibility that we'll make it so it still returns the dictionary card if elasticsearch times out or somesuch.